data/eos-metrics-event-recorder.service.in

```diff
@@ -1,6 +1,5 @@
 [Unit]
 Description=EndlessOS Metrics Event Recorder Server
-Wants=systemd-logind.service
```

Comment:
Also, normally, we might specify Wants since we do want it, sort of, but we also use DBus which will spawn it for us if it isn't already running, so it's fine. It's much cleaner to simply depend on DBus here.